kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriptionServiceImpl.java

```diff
@@ -102,83 +82,17 @@ public SubscriptionServiceImpl(@Qualifier("kilkariSettings") SettingsFacade sett
         this.subscriptionPackDataService = subscriptionPackDataService;
         this.subscriptionDataService = subscriptionDataService;
         this.subscriptionErrorDataService = subscriptionErrorDataService;
-        this.schedulerService = schedulerService;
         this.settingsFacade = settingsFacade;
         this.eventRelay = eventRelay;
         this.callRetryDataService = callRetryDataService;
         this.csrVerifierService = csrVerifierService;
-
-        schedulePurgeOfOldSubscriptions();
-    }
-
-
-    public Subscriber getSubscriber(final long callingNumber) {
-
-        SqlQueryExecution<Subscriber> queryExecution = new SqlQueryExecution<Subscriber>() {
-
-            @Override
-            public String getSqlQuery() {
-                return SELECT_SUBSCRIBERS_BY_NUMBER;
-            }
-
-            @Override
-            public Subscriber execute(Query query) {
-                query.setClass(Subscriber.class);
-                ForwardQueryResult fqr = (ForwardQueryResult) query.execute(callingNumber);
-                if (fqr.isEmpty()) {
-                    return null;
-                }
-                if (fqr.size() == 1) {
-                    return (Subscriber) fqr.get(0);
-                }
-                throw new IllegalStateException(String.format(MORE_THAN_ONE_SUBSCRIBER, callingNumber));
-            }
-        };
-
-        return subscriberDataService.executeSQLQuery(queryExecution);
-
+        this.allowMctsSubscriptions = true;
```

Comment:
actually nvm. its going to take insanely long to fix the tests that I wrote and redesign everything. With the current scheme of things, all instances will get reset to the same state @4am with a broadcast message. Saving this one for the next hotfix since this needs to be deployed in 12hrs.